### PR TITLE
(webdriver): re-export command

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Make what is documented [here](https://webdriver.io/docs/customcommands/#add-more-webdriver-commands) work when using CJS.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

Thanks @jlipps for reporting.

### CODE REVIEW

File: packages/webdriver/src/cjs/index.ts [hunk 1]
```diff
@@ -32,20 +39,19 @@ class WebDriver {
...
-    return async function protocolCommand(this: unknown, ...args: unknown[]) {
-        const commandESM = await import('../command.js')
-        return commandESM.default(method, encodeUri, commandInfo, doubleEncodeVariables).apply(this, args)
+    static get command () {
+        return command
     }
 }
+
+module.exports = WebDriver
+exports.command = command
```
Comment (jlipps) [inline packages/webdriver/src/cjs/index.ts:57]:
```suggestion
module.exports.command = command
```

Comment (christian-bromann) [reply packages/webdriver/src/cjs/index.ts:57]: `module.exports = WebDriver` which has `command` as static property, so it should be the same, right?

Comment (christian-bromann) [reply packages/webdriver/src/cjs/index.ts:57]: the test seem to pass:

```js
const WebDriver = require('../../packages/webdriver')
const command = require('../../packages/webdriver').command

assert.equal(typeof WebDriver.command, 'function')
assert.equal(typeof command, 'function')
```

Comment (jlipps) [reply packages/webdriver/src/cjs/index.ts:57]: oh I didn't see the refactor above, this was a suggestion based on the initial commit. let me try.

Comment (jlipps) [reply packages/webdriver/src/cjs/index.ts:57]: yep this should work now

Comment (christian-bromann) [reply packages/webdriver/src/cjs/index.ts:57]: 👍

**Pull request merged**